README.md

```diff
@@ -43,7 +43,7 @@ API documentation:
 To run a sample GraphQL server also serving GraphiQL, do the following:
 
 ```bash
-opam install graphql-lwt dune
+opam install graphql-lwt dune astring base64 cohttp cohttp-lwt-unix digestif ocplib-endian crunch
```

Comment:
Hmm, this list does not include `graphql-cohttp`, which is a dependency -- some of the other packages seem superfluous though.

I think this list should be sufficient:

```suggestion
opam install dune graphql-lwt graphql-cohttp cohttp-lwt-unix
```

If you could try if the above list works for you, it'd be great 🙏 😄